### PR TITLE
Documentation for new feature MAS table macro

### PR DESCRIPTION
**Pull request by DVTimWilson**
Documentation for new feature multi-active satellite table macro.

### CODE REVIEW

File: docs/macros.md [hunk 1]
```diff
@@ -566,23 +567,24 @@ Generates sql to build a satellite table using the provided parameters.
...
+
         records_to_insert AS (
-            SELECT DISTINCT e.CUSTOMER_PK, e.HASHDIFF, e.CUSTOMER_NAME, e.CUSTOMER_PHONE, e.CUSTOMER_DOB, e.EFFECTIVE_FROM, e.LOAD_DATE, e.SOURCE
-            FROM source_data AS e
+            SELECT DISTINCT stage.CUSTOMER_PK, stage.HASHDIFF, stage.CUSTOMER_NAME, stage.CUSTOMER_PHONE, stage.CUSTOMER_DOB, stage.EFFECTIVE_FROM, stage.LOAD_DATE, stage.SOURCE
+            FROM source_data AS stage
             LEFT JOIN latest_records
-            ON latest_records.HASHDIFF = e.HASHDIFF
-            WHERE latest_records.HASHDIFF IS NULL
+                ON latest_records.CUSTOMER_PK = stage.CUSTOMER_PK
```
Comment (DVAlexHiggs) [inline docs/macros.md:583]: Tab this correctly, align with other SQL above.

File: docs/tutorial/tut_multi_active_satellites.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    ```
+
+To create a MAS model, we simply copy and paste the above template into a model named after the MAS we
+are creating. dbtvault will generate a MAS using parameters provided in the next steps.
+
+MAS should use the incremental materialization, as we load and add new records to the existing data set. 
+
+We recommend setting the `incremental` materialization on all of your MAS using the `dbt_project.yml` file:
+
+=== "dbt_project.yml"
```
Comment (DVAlexHiggs) [inline docs/tutorial/tut_multi_active_satellites.md:74]: We need to update this section in ALL of the tutorial steps not to include the `dbt_project.yml` example anymore. Might be best to link to the Metadata reference, with a short explanation in the tutorial step.

File: docs/tutorial/tut_multi_active_satellites.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+3. A hashdiff. We created `HASHDIFF` in [staging](tut_staging.md) earlier, which we will use here.
+4. Some payload columns: `CUSTOMER_NAME`, `CUSTOMER_PHONE` which should be present in the 
+raw staging layer via an [stage](../macros.md#stage) macro call.
+5. An `EFFECTIVE_FROM` column, also added in staging. 
+6. A load date timestamp, which is present in the staging layer as `LOAD_DATE`. 
+7. A `SOURCE` column.
+
+We can now add this metadata to the `dbt_project`:
+
+=== "dbt_project.yml"
```
Comment (DVAlexHiggs) [inline docs/tutorial/tut_multi_active_satellites.md:138]: We need to update this section in ALL of the tutorial steps to not to include the dbt_project.yml example anymore. Might be best to link to the Metadata reference, with a short explanation in the tutorial step.

File: docs/macros.md (hunk 1 above)
Comment (DVTimWilson) [reply docs/macros.md:583]: This is a change in the sat section of the documentation, not mas. The example sql here for sat in fact has also not been updated to reflect the current version of sat.sql, and sat.sql itself has a layout issue in this section. So both the documentation and the feature code for sat need minor corrections. Perhaps as this is a MAS PR this should be allowed through as is and the corrections made as bug fixes through future PRs?

File: docs/tutorial/tut_multi_active_satellites.md (hunk 3 above)
Comment (DVTimWilson) [reply docs/tutorial/tut_multi_active_satellites.md:138]: This again seems like a task for a future bug fix PR including the required changes for all tutorials?

Comment (DVAlexHiggs) [reply docs/tutorial/tut_multi_active_satellites.md:138]: Yes agreed

**Review by DVAlexHiggs — APPROVED** [review]
Approved. I'll create a new feature branch for the changes discussed which aren't MAS specific.

**Pull request merged**